content/en/seameet/seameet-manual/2-general-setup/03-seameet-how-to-invite-seameet.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Transcribe Meetings"
+description: ""
+date: 2023-11-22T08:48:57+00:00
+lastmod: 2023-11-22T08:48:57+00:00
+draft: false
+images: []
+menu:
+  seameet:
+      parent: "03-seameet-how-to-invite-seameet"
+aliases:
+   - /zh/seameet/03-seameet-how-to-invite-seameet/
+weight: 21
+toc: true
+---
+
+# Invite SeaMeet Copilot to Transcribe Meetings
+
+SeaMeet is an AI tool that automatically generates meeting transcripts by inviting the copilot into the meeting room. Currently, SeaMeet mainly supports the Google Meet platform. SeaMeet supports multiple invitation methods, and you can choose one of the following three methods to invite the copilot:
+
+### 1. Workspace Invitation
+
+1. Click the **Start Recording** button.
+2. Enter the Google Meet meeting code and submit.
+3. The meeting will automatically be added to the meeting list.
+
+<center>
+<img src="/images/seameet-en/03-seameet-how-to-invite-seameet/workspace-invitation.png" alt="SeaMeet Workspace Invitation"/>
+</center>
+
+### 2. SeaMeet Extension Invitation
+
+1. Install the extension from the Chrome Web Store (also available on Edge).
+
+   [https://chrome.google.com/webstore/detail/seameet-ai-meeting-minute/gkkhkniggakfgioeeclbllpihmipkcmn](https://chrome.google.com/webstore/detail/seameet-ai-meeting-minute/gkkhkniggakfgioeeclbllpihmipkcmn)
+
+2. After installation, open Google Meet, and the extension will automatically open.
+
+<center>
+<img src="/images/seameet-en/03-seameet-how-to-invite-seameet/extension-invitation.png" alt="SeaMeet Extension Invitation"/>
+</center>
+
+3. Click **Start Recording** to invite the copilot to the meeting.
+4. Allow the copilot to join and start recording the meeting.
+
+<center>
+<img src="/images/seameet-en/03-seameet-how-to-invite-seameet/exntension-transcribing.png" alt="Chrome Extension Transcribing"/>
+</center>
+
+### 3. Google Calendar Invitation
+
+1. Invite the SeaMeet copilot <email: meet@seasalt.ai> to the meeting in Google Calendar.
+2. The SeaMeet copilot will join the Google Meet meeting on time.
+
+<center>
```

Comment:
this is what i see: 
![image](https://github.com/seasalt-ai/wiki/assets/543547/d1d4394d-db1c-4ee0-bc21-545b4669e103)

did you check the PR when you submitted it for review?